packages/configuration-development/package.json

```
@@ -17,6 +17,7 @@
     "@hint/hint-meta-charset-utf-8": "^4.0.3",
     "@hint/hint-meta-viewport": "^5.0.3",
     "@hint/hint-no-bom": "^4.2.7",
+    "@hint/hint-no-inline-styles": "^1.0.0",
```

Looks like I missed this on your original PR, but we should change the version number in all the references and in the `package.json` for `hint-no-inline-styles` to `0.0.1`. 

Our release script expects something less than `1.0.0` as part of calculating the initial changelog when a package hasn't been published yet. The script will then bump it to `1.0.0` as part of the release.

```suggestion
    "@hint/hint-no-inline-styles": "^0.0.1",
```

---

Thanks, @antross, I just updated it!